app/controllers/applications_controller.rb

```diff
@@ -3,17 +3,14 @@ class ApplicationsController < ApplicationController
   def show
     @application = Application.find(params[:id])
     @pets = @application.pets
-    @pets_search = Pet.where(name: params[:search])
-  end
-
-  def application_params
-    params.permit(:name, :address, :city, :state, :zip, :description)
+    @pets_search = search_by_name(params[:search])
```

Comment:
I think I messed up and we should change the params[:search] to search_params.

Reply:
Now looking at the below comment, maybe we change it to `@pets_search = search_by_name(search_params[:search])`

Reply:
Well jeeze, now I think we could get rid of the search_by_name method all together and just have `@pets_search = Pets.search(search_params[:search])`